package.json

```diff
@@ -65,6 +65,7 @@
     "later": "^1.2.0",
     "lodash": "^4.17.10",
     "lodash.pick": "^4.4.0",
+    "magnificent": "^2.2.3",
```

Comment:
Nice work so far.
Can I know what this module is used for?, I don't see it being used in any of the files.